hydrolib/core/dflowfm/ext/models.py

```diff
@@ -181,6 +184,117 @@ def validate_location_type(cls, v: str) -> str:
         return v
 
 
+class MeteoForcingFileType(str, Enum):
+    """
+    Enum class containing the valid values for the forcingFileType
+    attribute in Meteo class.
+    """
+
+    bcascii = "bcAscii"
+    """str: Space-uniform time series in <*.bc> file."""
+
+    netcdf = "netcdf"
+    """str: NetCDF, either with gridded data, or multiple station time series."""
+
+    uniform = "uniform"
```

Comment:
To check: I do not see this option in the manual

Reply:
I have fixed this now in the User Manual.